docs/src/main/asciidoc/configuration.adoc

```diff
@@ -51,37 +51,46 @@ There are some properties that could be shared among different Azure services, f
 
 NOTE: Properties configured under each Azure service will override the global configurations.
 
-Spring Cloud Azure unifies configuration properties' prefixes to `spring.cloud.azure` since 4.0, which will make configuration properties more consistent and more intuitive. Here's a quick review of the prefixes of supported Azure services.
+Spring Cloud Azure unifies configuration properties' prefixes to `spring.cloud.azure` since 4.0, which will make configuration properties more consistent and more intuitive. Here's a quick review of the serivce specific properties.
 
 .Service specific properties
-[cols="<50,<50", options="header"]
+[cols="<3,<3,<3", options="header"]
```

Comment:
I am thinking about if we can add JMS related properties in this table or not, but given those are not prefixed with `spring.cloud.azure`, so perhaps we can introduce the jms property link with another paragraph after the table?

Reply:
this is for the level one sdk configuration, so the jms won't be included in this table. That's why we have a whole section for JMS.